jenkinsbox.yml

```diff
@@ -71,6 +64,11 @@
       - apache
 
   # @tag environment.
+  - name: Set Jenkins server IP
+    set_fact:
+      jenkins_server_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
this should be tested ion VM as well, because there were issues with getting IP in a such manner previously